policy/modules/services/ntp.te

```diff
@@ -171,6 +171,7 @@ ifdef(`init_systemd',`
 	optional_policy(`
 		chronyd_enabledisable(ntpd_t)
 		chronyd_startstop(ntpd_t)
+		chronyd_status(ntpd_t)
```

Comment:
Related to the comment about cockpit_enabledisable and cockpit_startstop granting 'status' access, should this be here, or should I update chronyd_enabledisable and chronyd_startstop to allow 'status' access instead?

Reply:
> should I update chronyd_enabledisable and chronyd_startstop to allow 'status' access instead?

Yes, this.

Reply:
updated